site-main/main.tf

```diff
@@ -158,5 +158,70 @@ resource "aws_cloudfront_distribution" "website_cdn" {
       Project     = var.project
     },
   )
+
+  dynamic "origin" {
+    for_each = [for o in var.origins : {
+      origin_id                        = o.origin_id
+      domain_name                      = o.domain_name
+      origin_path                      = o.origin_path
+      origin_protocol_policy           = o.origin_protocol_policy
+      duplicate_content_penalty_secret = o.duplicate_content_penalty_secret
+    }]
+    content {
+      origin_id   = origin.value.origin_id
+      domain_name = origin.value.domain_name
+      origin_path = origin.value.origin_path
+
+      custom_origin_config {
+        origin_protocol_policy = origin.value.origin_path
+        http_port              = "80"
+        https_port             = "443"
+        origin_ssl_protocols   = ["TLSv1"]
```

Comment:
Can we make this `TLSv1.2` without anything breaking?

Reply:
I can't see anything obvious breaking. Let's bump it and give it a go